### DISCUSSION TITLE
Unable to squash and merge PR from mobile

### DISCUSSION THREAD

**Discussion by andr-ll** [community/community#73753] [Mobile] [answered]
### Select Topic Area

Bug

### Body

Good day all!

I am facing an issue with ability to `Squash and Merge` PRs from mobile app.

1. First of all, before it was `Squash and Merge` default button - now it is `Rebase and Merge`

<img src="https://github.com/community/community/assets/72939898/7f4ead53-4c9b-4330-bb2d-4baa367f8844" height=500>

2. If it’s pressed following error appears

<img src="https://github.com/community/community/assets/72939898/d91b7fa1-d915-4684-8cf4-709a632b716c" height=500>

3. Then when it’s changed to `Squash and merge` and pressed again - same error appears

<img src="https://github.com/community/community/assets/72939898/8753a840-f855-4864-a500-e7333e05b3df" height=500>

<img src="https://github.com/community/community/assets/72939898/292000c9-d750-4dcc-8eac-27b1b7500e04" height=500>

On web it works as it was working before.

Thanks in advance for the fix :)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.137.1 (114555847)
Device: iPhone Xs (iOS 16.6.1)
</details>

**Comment by codewithriza** [comment]
To fix this:

Clear app cache and data.
Update the app.
If the problem continues, reinstall the app.
Contact app support, and share the debug info if possible.
Check if the app's recent changes caused this issue.

**Comment by serzmerz** [comment]
I have the same issue, looks like a bug

**Comment by jmeridth** [comment]
This bug still exists. Tried everything suggested. Response in https://github.com/orgs/community/discussions/65461 hints it is a known bug. Awaiting released fix.

**Reply by andr-ll** [reply, marked as answer]
I just updated to new version yesterday (v1.137.2, iOS) and it works fine :)

**Reply by jmeridth** [reply]
Confirmed.  Working for me on latest beta 139.0
